src/main/java/com/quizapp/enterprise/models/User.java

```diff
@@ -12,7 +12,7 @@
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private int userID;
```

Comment:
This one is `userID` and everything else is `someId` so you should change this to keep it consistent

Reply:
Good catch.